src/components/Member.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import Jocelyn from './imgs/jocelyn.jpg';
+
+class Member extends Component {
```

Comment:
Would this be better as a functional component since it is only displaying text and not handling button clicks or anything?